### PR TITLE
Updated Profile, Create Account, Sign Up

### PR DESCRIPTION
**Pull request by tkim516**
### Pull Request Summary
Updated designs for profile, create account, sign up, forgot password components/pages
Added zipcode prop to userSchema and corrected typo in receiveNewsletetr prop

### Pull Request Checklist

- [ ] Code is neat, readable, and works
- [ ] Comments are appropriate
- [ ] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [ ] The developer name is specified
- [ ] The summary is completed
- [ ] Assign reviewers

### Screenshots
Screenshot 2024-05-23 at 1 34 38 PM

Screenshot 2024-05-23 at 1 34 49 PM

Screenshot 2024-05-23 at 1 35 00 PM

Screenshot 2024-05-23 at 1 36 09 PM

Screenshot 2024-05-23 at 2 30 08 PM

Screenshot 2024-05-23 at 2 30 16 PM

### CODE REVIEW

File: src/app/signup/signupComponent.tsx [hunk 1]
```diff
@@ -45,20 +51,6 @@ export default function SignUp() {
   const [passwordError, setPasswordError] = useState(false);
   const [emailError, setEmailError] = useState(false);
 
-  useEffect(() => {
```
Comment (javalosr2004) [inline src/app/signup/signupComponent.tsx:48]: why was this removed? Shawn just implemented it.

**Review by javalosr2004 — CHANGES_REQUESTED** [review]
Aside from the question of removing the useEffect that parses session storage, everything looks great! Nice refactor, can't have misspelled variables in production code, likewise with the font change to Lato.

File: src/app/signup/signupComponent.tsx (hunk 1 above)
Comment (tkim516) [reply src/app/signup/signupComponent.tsx:48]: My bad, I think it got removed when I merged?

Comment (tkim516) [reply src/app/signup/signupComponent.tsx:48]: Will add it back

**Review by javalosr2004 — APPROVED** [review]
